### ISSUE TITLE
Error running admin project

### ISSUE THREAD

**Issue by VerbindApp** [strapi/strapi#1036]
**Informations**
- **Node.js version**:  9.8.0
- **npm version**: 5.6.0
- **Strapi version**: 3.0.0-alpha.12.0.1
- **Database**: postgres
- **Operating system**: Windows 8.1

**What is the current behavior?** When i try to run only admin project it shows this error npm ERR! strapi-admin@3.0.0-alpha.12.0.1 start: `node ./node_modules/strapi-helper-plugin/node_modules/.bin/cross-env NODE_ENV=development POR_modules/strapi-helper-plugin/lib/server`

if i modify the script of star tlike this
    "start": "node ./node_modules/.bin/cross-env NODE_ENV=development PORT=4000 IS_ADMIN=true node ./node_modules/strapi-helper-plugin/lib/server",

it shows me this error

> node ./node_modules/.bin/cross-env NODE_ENV=development PORT=4000 IS_ADMIN=true node ./node_modules/strapi-helper-plugin/lib/server

module.js:545
    throw err;
    ^

Error: Cannot find module 'C:\Scotia\beyond\cms\strapi12\admin\node_modules\.bin\cross-env'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:470:25)
    at Function.Module.runMain (module.js:690:10)
    at startup (bootstrap_node.js:194:16)
    at bootstrap_node.js:666:3
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! strapi-admin@3.0.0-alpha.12.0.1 start: `node ./node_modules/.bin/cross-env NODE_ENV=development PORT=4000 IS_ADMIN=true node ./node_modules/strapi-helper-plugin/lib/server`

**Steps to reproduce the problem**.

Create a new project.
Go to admin folder
execute npm start

**What is the expected behavior?**
Run the admin project.

**Suggested solutions**

- [ ] I'm sure that this issue hasn't already been referenced

**Comment by soupette** [comment]
You have to start your project in dev mode to do that. Please refer to the [PLUGIN DEVELOPMENT section](https://strapi.io/documentation/plugin-development/quick-start.html).

**Comment by patrikniebur** [comment]
I'm having the same issue and I followed the documentation. Was able to resolve it by running:
`cd ./node_modules/strapi-helper-plugin/ && npm install`

Then it starts admin on port 4000 (specified in package.json) while trying to load other resources from port 1337 (specified in server.json). When I make those consistent it still fails on loading non-json file as json

**Comment by krsnaa** [comment]
run `npm link strapi-helper-plugin` from the admin folder, then run `npm start`

**Comment by lauriejim** [comment]
@knappsacks can you give us your OS and the version of Strapi you use.

**Comment by knappsacks** [comment]
@lauriejim strapi version: `3.0.0-alpha.21`; OS: `macOS High Sierra 10.13.6`.

I think it's possibly my fault, however. It appeared on the [Admin Panel Strapi Documentation Page](https://strapi.io/documentation/3.x.x/advanced/customize-admin.html#files-structure) that I could just spin up the admin panel locally and make some changes. However, I realized per the contributing guidelines it's actually a bit more involved and I needed to clone the initial repo rather than just trying to alter the NPM install - [found here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md).
